gg-pac-team3-front/src/pages/frontPage/FrontPage.jsx

```diff
@@ -0,0 +1,12 @@
+import NavBlue from "../../components/layout/navBlue";
+import Panel from "./Panel";
+
+function FrontPage(){
```

Comment:
I would expect that our NavBar should appear on every page, not just our front page. The contents of the bar we can change in the future depending on the logged in user. Could we try using an `Outlet` here for this (which would probably not go in this file, but this file can just be the actual content of the front page).

https://reactrouter.com/en/main/components/outlet